docs/pages/tutorial/eas/internal-distribution-builds.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Create and share internal distribution build
+sidebar_title: Internal distribution build
+description: Learn about what are internal distribution builds, why we need them, and how to create them.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import { CODE } from '~/ui/components/Text';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, we're going to learn how to set up [internal distribution builds](/guides/sharing-preview-releases/#internal-distribution). Creating an internal build is quite similar to creating a development build. However, the difference is how it is used and distributed.
+
+## Internal distribution build
+
+An internal distribution build is great for sharing an app's progress with our team (technical and non-technical stakeholders) to gather feedback without running a development server. Our team can directly install the build and run it like an app on their device.
+
+## Ways to distribute an app internally
+
```

Comment:
Seems like there should be a sentence here about how both Apple and Google have built-in mechanisms for sharing an app internally etc.